test-apps/ui-test-app/src/frontend/index.tsx

```diff
@@ -496,14 +500,40 @@ export class SampleAppIModelApp {
 
       const iTwin: ITwin = iTwinList[0];
 
-      const iModel = (await (new IModelHubClient()).iModels.get(accessToken, iTwin.id, new IModelQuery().byName(iModelName)))[0];
+      const toAuthorization = (token: AccessToken): Authorization => {
+        const splitAccessToken = token.split(" ");
+        if (splitAccessToken.length !== 2)
+          throw new Error("Unsupported access token format");
+
+        return {
+          scheme: splitAccessToken[0],
+          token: splitAccessToken[1],
+        };
+      };
+
+      if (!SampleAppIModelApp.hubClient)
+        return;
+
+      let iModel;
+      for await (const imodel of SampleAppIModelApp.hubClient.iModels.getRepresentationList({
+        urlParams: {
+          name: iModelName,
+          projectId: iTwin.id,
+          $top: 1,
+        },
+        authorization: async () => toAuthorization(accessToken),
+      }))
+        iModel = imodel;
```

Comment:
Same remark as for this bit - https://github.com/iTwin/itwinjs-core/blob/63e800b8eae4e785893776ef98292aa2515464ae/test-apps/ui-test-app/src/frontend/appui/imodelindex/IModelIndex.tsx#L118

 $top is probably not enough

Reply:
I see you're using a name filter so it's fine.